modules/codemod/lib/v6/spec/moveAndRenameSearchBar.spec.ts

```diff
@@ -78,18 +78,73 @@ describe('Canvas Kit Move and Rename Search Bar Codemod', () => {
 
       expectTransform(input, expected);
     });
+
+    it('should ignore non-canvas-kit components from main package', () => {
+      const input = `
+        import { SearchBar as CanvasSearchBar } from "@workday/canvas-kit-labs-react";
+        import { SearchBar } from "@other/search-bar";
+
+        const CustomSearch = () => {
+          return <>
+            <CanvasSearchBar />
+            <SearchBar />
+          </>;
+        }
+      `;
+      const expected = `
+        import { SearchForm as CanvasSearchBar } from "@workday/canvas-kit-labs-react";
+        import { SearchBar } from "@other/search-bar";
+
+        const CustomSearch = () => {
+          return <>
+            <CanvasSearchBar />
+            <SearchBar />
```

Comment:
This is a less likely case than Header, but it's possible we have a similarly named non-Canvas Kit SearchBar. I added support for this just in case.